packages/app/src/app/pages/Dashboard/Sidebar/index.js

```diff
@@ -135,12 +135,7 @@ class Sidebar extends React.Component {
         </Route>
 
         <div style={{ margin: '2rem', fontSize: '.875rem' }}>
-          <Button
-            style={{ display: 'block' }}
```

Comment:
Shouldn't we need to keep `display: 'block'` here?

Reply:
No because it displays the same with that extra style being set or not. The `block` prop ensures that in this particular instance, the button fills 100% of the width of it's container. I think this would only matter if the underlying element was an inline element, such as an `a` tag, which `block` also solves, as it sets the display of the element to `flex` versus `inline-flex`.